tests/spec/puppeteer-axe-spec.js

```diff
@@ -0,0 +1,60 @@
+/* Puppeteer tests for the A11y. */
+const { AxePuppeteer } = require('axe-puppeteer');
+const puppeteer = require('puppeteer');
+
+describe("Puppeteer accessibility tests for the Python Editor.", function() {
+    'use strict';
+
+    const editorURL = 'http://localhost:5000/editor.html';
```

Comment:
We should do help.html as well.

Reply:
Done in https://github.com/bbcmicrobit/PythonEditor/pull/302/commits/e9dfb958dc9ece81d97e9e6118bdd7f33d6aa094

Reply:
Done in https://github.com/bbcmicrobit/PythonEditor/pull/302/commits/e9dfb958dc9ece81d97e9e6118bdd7f33d6aa094